### DISCUSSION TITLE
any plans to support osx? thx

### DISCUSSION THREAD

**Discussion by kocoman1** [cyring/CoreFreq#360] [Ideas]
any plans to support osx? thx

**Comment by cyring** [comment]
> any plans to support osx? thx

I would like to, but:
1. Don't have a Mac
2. kext driver may required certificate
3. Apple has moved to M1, M2. No more x86 in near future ?

**Comment by cyring** [comment]
> any plans to support osx? thx

Now that I have ported _CoreFreq_ to arm64, I plane to acquire an Apple Silicon machine:at least a cheap/used M1 MacBook.
